src/toniarts/openkeeper/world/creature/steering/AbstractCreatureSteeringControl.java

```diff
@@ -105,19 +102,27 @@ protected void applySteering(SteeringAcceleration<Vector2> steering, float tpf)
         // TODO: Call function?
         if (steering.isZero()) {
             steeringBehavior = null;
+            stopMoving = false;
+            return;
         }
 
         // Update orientation and angular velocity
-        if (independentFacing) {
-            setOrientation(getOrientation() + (angularVelocity * tpf));
-            angularVelocity += steering.angular * tpf;
+        if (!stopMoving) {
+            position.add(linearVelocity.x * tpf, linearVelocity.y * tpf);
+            linearVelocity.mulAdd(steering.linear, tpf).limit(maxLinearSpeed);

```

Comment:
Main idea:
-add FollowPath and Face steering
- creature move to target
- when moving stops (end point is reached), creature rotates around Y-axis (faced).

But on my other computer with downloaded gdx-ai library it feel different with other computer.